addons/hr_holidays/report/hr_leave_report_calendar.py

```diff
@@ -81,7 +81,7 @@ def _fetch_query(self, query, fields):
         records = super()._fetch_query(query, fields)
         if self.env.context.get('hide_employee_name') and 'employee_id' in self.env.context.get('group_by', []):
             self.env.cache.update(records, self._fields['name'], [
-                record.name.split(':')[-1].strip()
+                (record.name.split(':')[-1].strip() if record.name else record.name)
```

Comment:
```suggestion
                (record.name.split(':')[-1].strip() if record.name else '')
```
What do you think of that ? I was a confused by the else value :)

Reply:
i think it will just give false value but this could work as well Thanks

Reply:
you can just put False too; just just the same value was a bit confusing for me :)